.gitignore

```diff
@@ -12,6 +12,9 @@ mvn_test.out
 # used when running ./emissary script
 .mvn-classpath
 
+# used when running ./Docker_compose_test_script.sh
+test_results
```

Comment:
consider moving this to target/test_results

Reply:
Fixed test_results output as suggested.